cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml

```diff
@@ -89,10 +99,8 @@ spec:
           - --target-port=31337
           - --stackdriver-prefix={{ prometheus_to_sd_prefix }}/addons
           - --api-override={{ prometheus_to_sd_endpoint }}
+          - --source=
```

Comment:
Why this change?

Reply:
Oo, stray change. Thanks for catching this!